packages/web-client/app/components/card-space/create-space-workflow/index.ts

```diff
@@ -293,7 +293,7 @@ class CreateSpaceWorkflow extends Workflow {
       errors.push(FAILURE_REASONS.RESTORATION_UNAUTHENTICATED);
     }
 
-    // TODO: check if there are any business accounts available for Card Space which haven't been used yet
+    // TODO: check if there are any payment profiles available for Card Space which haven't been used yet
```

Comment:
Should we just delete this file already??? So we don’t keep updating it for no reason [CS-3326](https://linear.app/cardstack/issue/CS-3326/delete-create-card-space-workflow)